xbmc/filesystem/test/TestFile.cpp

```diff
@@ -103,7 +103,7 @@ TEST(TestFile, Write)
   char buf[30];
   memset(&buf, 0, sizeof(buf));
 
-  ASSERT_TRUE((file = XBMC_CREATETEMPFILE("")) != NULL);
+  ASSERT_NE(nullptr, file = XBMC_CREATETEMPFILE(""));
```

Comment:
there are parantheses around all other assignments ... seems only this one slipped through?